docs/src/modules/utils/defaultPropsHandler.js

```diff
@@ -0,0 +1,96 @@
+import recast from 'recast';
+import { utils as docgenUtils } from 'react-docgen';
+// https://github.com/reactjs/react-docgen/pull/351
+import isReactForwardRefCall from 'react-docgen/dist/utils/isReactForwardRefCall';
```

Comment:
Seems to be released in https://github.com/reactjs/react-docgen/releases/tag/v5.0.0-beta.1